packages/@aws-cdk/toolkit-lib/test/actions/deploy.test.ts

```diff
@@ -53,10 +54,9 @@ describe('deploy', () => {
 
   test('request response when require approval is set', async () => {
```

Comment:
this test tests that we are getting the approval prompt... do you also want me to test that we can respond to it? or is that self-explanitory

Reply:
I think that's enough. Otherwise we're just testing that the `TestIoHost` works.

I'd add a `data` prop to the expected message though.